eloquentjs/2_2_fizzbuzz.js

```diff
@@ -0,0 +1,26 @@
+/** Performs the FizzBuzz operation from 1 to iter(inclusive)*/
```

Comment:
Same comments about the docstring formatting, no return after throw, and using vertical white space to clarify intent.